### PR TITLE
Blog - Securing Kubernetes Secrets Effectively

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
Preview URL: https://6408d70e29a7fb0c5f407cdb--ephemeral-narwhal-3ac3b2.netlify.app

File: blog/_posts/2023-04-19-securing-kubernetes-secrets.md [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+
+The parameters in the command stand for the following:
+
+- **`req`**: This parameter tells OpenSSL to generate a new CSR.
+- **`-new`**: This option generates a new CSR.
+- **`-key john.key`**: This parameter specifies the private key to use when generating the CSR. Here, the private key is stored in the file named "john.key".
+- **`-out john.csr`**: This option specifies the filename of the output CSR. The resulting CSR will be stored in the file named "john.csr".
+- **`-subj "/CN=john/O=marketing"`**: This parameter specifies the subject of the CSR. The **`/CN=john`**  part of the subject shows the Common Name (CN) of the entity that the certificate will be issued to. The **`/O=marketing`** part of the subject specifies the name of the organization that the entity belongs to.
+
+Copy the **ca.crt* and *ca.key* securely (SCP) from the `/etc/kubernetes/pki` directory from your master node using the below command, you will need the *ca.crt* and *ca.key* on your local machine to sign John's certificate:
```
Comment (balapriyac) [inline blog/_posts/2023-04-19-securing-kubernetes-secrets.md:303]:
```suggestion
Copy the *ca.crt* and *ca.key* securely (SCP) from the `/etc/kubernetes/pki` directory from your master node using the below command, you will need the *ca.crt* and *ca.key* on your local machine to sign John's certificate:
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Thanks, Sameer. Looks great. Requested for one small change.

**Comment by msali123** [comment]
Thanks Bala, changes done

Preview URL: https://6410358d9a3699619906c385--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by balapriyac** [comment]
@jalletto this is ready to merge, too.

**Review by jalletto — APPROVED** [review]

**Pull request merged**